packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -19,11 +19,14 @@ export interface InterventionPolicyGroup {
 	name: string;
 	parameter: string;
```

Comment:
I think we should remove this parameter. 
This is the same as intervention.appliedTo

Reply:
This will be done by me in future PR